app/components/PageTitle.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import React from 'react';
+
+interface PageTitleProps {
+  subtitle1?: string;
+  subtitle2?: string;
+  subtitle3?: string;
+  subtitleClass?: string;
+  title?: string;
+}
+
+const PageTitle = (props: PageTitleProps) => {
```

Comment:
Just a suggestion, I would keep components definition the same way you did below in PreContentText

```jsx
export default function PageTitle(props: PageTitleProps) {
  return (
    ...
  );
}
```